Userland/Libraries/LibGUI/Splitter.cpp

```diff
@@ -222,11 +224,27 @@ void Splitter::mousemove_event(MouseEvent& event)
     }
 
     if (m_orientation == Orientation::Horizontal) {
-        m_first_resizee->set_fixed_width(fixed_resizee() == FixedResizee::First ? new_first_resizee_size.width() : -1);
-        m_second_resizee->set_fixed_width(fixed_resizee() == FixedResizee::Second ? new_second_resizee_size.width() : -1);
+        if (fixed_resizee() == FixedResizee::First) {
```

Comment:
Why does `fixed_resizee` have to `e`s at the end (i.e. why is not `fixed_resize`)?